webpack.config.base.js

```diff
@@ -21,7 +21,6 @@ module.exports = {
   resolve: {
     alias: {
       'react': specificReact,
-      'react-dom': '@hot-loader/react-dom',
```

Comment:
Does this impact the DLL stuff, seems like an unrelated change? Does hot loader still work in dev if you remove this alias?

Reply:
Good question. I'll test it out and if it is needed for dev, I can move it into `webpack.config.cli.dev.js`.

Reply:
Sure enough, without the alias you get this warning in dev mode: `React-Hot-Loader: react-dom patch is not detected. React 16.6+ features may not work.`

I added it back into `webpack.config.cli.dev.js` so it will only apply to dev builds and I confirmed that the warning disappears.